lib/subscriptionClient.ts

```diff
@@ -234,4 +240,37 @@ export class SubscriptionClient extends Client {
     return this._context.managementClient!.receiveDeferredMessages(sequenceNumbers,
       this.receiveMode);
   }
+
+  //#region topic-filters
+
+  /**
+   * Get all the rules associated with the subscription
+   */
+  async getRules(): Promise<RuleDescription[]> {
+    return this._context.managementClient!.getRules();
+  }
+
+  /**
+   * Removes the rule on the subscription identified by the given rule name.
+   * @param ruleName
+   */
+  async removeRule(ruleName: string): Promise<void> {
+    return this._context.managementClient!.removeRule(ruleName);
+  }
+
+
+  /**
+   * Adds a rule on the subscription as defined by the given rule name, filter and action.
+   * Remember to remove the default true filter on the subscription before adding a rule,
+   * otherwise, the added rule will have no affect as the true filter will always result in
+   * the subscription receiving all messages.
+   * @param ruleName Name of the rule
+   * @param filter A Boolean, SQL expression or a Correlation filter
+   * @param sqlRuleActionExpression Action to perform if the message satisfies the filtering expression
```

Comment:
Since the expression is of type `string`, the customer would have no idea of what would constitute as a valid sql expression. Providing some examples of sqlRuleActionExpression  in a bullet list fashion and a link to the [doc](https://docs.microsoft.com/en-us/azure/service-bus-messaging/topic-filters#actions) would improve the  developer experience.

Reply:
Added links to docs